File: eel_v/web/backend/table_instance.py
from web.backend.local.dbOperations import DataBase
from mysql.connector.errors import OperationalError, ProgrammingError

import eel

class Instance:#для выовда селекта
    def __init__(fels, db : DataBase, command):
        Instance.db = db
        Instance.result = db.make_command(command, False)
        
    @eel.expose
    def show_r():
        if eel.btl.request.get_cookie('is_connected') == None:
            eel.go_to_connect()
        else:
            pass
        
        if type(Instance.result) == OperationalError or type(Instance.result) == ProgrammingError:
            eel.show_error()
            return 1
        else:
            eel.load_tables_inst(Instance.result)

File: eel_v/web/backend/javascript/table_instance.js
eel.show_r()

eel.expose(load_tables_inst)
function load_tables_inst(result){

    let table = document.querySelector('table')

    for (let i = 0; i < result.length; i++){

        let tr = document.createElement('tr')
        tr.id = 'row'

        for (let j = 0; j < result[i].length; j++){

            let td = document.createElement('td')

            let em = document.createElement('em')
            em.innerHTML = result[i][j]

            td.appendChild(em)
            tr.appendChild(td)

        }

        table.appendChild(tr)

    }

}